docs/src/create_reference_md.py

```diff
@@ -142,6 +145,8 @@ def source_section(source: Source, source_key: str, level: int) -> str:
     """Return the source as Markdown section."""
     markdown = markdown_header(source.name, level, index=True)
     markdown += markdown_paragraph(source.description)
+    if documentation := (source.documentation and source.documentation.get("generic")):
+        markdown += markdown_paragraph(documentation)
```

Comment:
How/where is non-generic documentation displayed in the docs?

Reply:
Added a comment.